gorm/user.go

```diff
@@ -142,37 +142,54 @@ func GetUsersForAdmin(fn string, gususername string, sortBy string) ([]apiObject
 	switch sortBy {
 	case "idDESC":
 		sortBy = "u.id DESC"
-		logger.Info(sortBy)
 	case "full_nameDESC":
 		sortBy = "u.full_name DESC"
-		logger.Info(sortBy)
 	case "gus_usernameDESC":
 		sortBy = "u.gus_username DESC"
-		logger.Info(sortBy)
 	}
 	sql.WriteString(`SELECT
 			u.*,
 			(SELECT COUNT(*)
 			FROM orders o_sub
 			WHERE o_sub.user_id = u.id
-			AND o_sub.status_code >= ?) AS total_orders
+			AND o_sub.status_code >= ?) AS total_orders,
+            (SELECT COUNT(*) 
+            FROM redeemable_codes r
+            WHERE r.redeemed_by_user_id = u.id
```

Comment:
Please indent these lines equal to the lines above and below

Reply:
I did :v